include/library/spdm_common_lib.h

```diff
@@ -214,10 +216,14 @@ typedef enum {
 
 /* The Endian values apply only if the spdm version is 1.0/1.1.
  * The default verification mode is big endian only. */
-#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_ONLY        (0)
-#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_LITTLE_ONLY     (1)
-#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_OR_LITTLE   (2)
-
+#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_ONLY 0
+#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_LITTLE_ONLY 1
+#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_OR_LITTLE 2
+
```

Comment:
/* The Endian values apply only if the secured spdm version is 1.0/1.1.
 * The default sequence number mode is enc little and dec little endian only. */